.github/workflows/main.yml

```diff
@@ -47,7 +47,8 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: '3.11'
+          architecture: 'x64'
```

Comment:
why do we need to specify the architecture here?

Reply:
We don't need to, but the architecture is always specified everywhere else where python-version is specified. This was the only place where it was missing.

Reply:
I would remove this and let githubs runner decide which architecture is available as we don't have a requirement for this :shrug: